plot_srt_stats.py

```diff
@@ -393,7 +417,8 @@ def plot_graph(stats_filepath, is_sender, is_fec, export_png):
     plots['window_size'] = create_window_size_plot(source)
 
     # Latency
-    plots['latency'] = create_latency_plot(source)
+    #plots['latency'] = create_latency_plot(source)
```

Comment:
Is the latency plot not actual anymore? The function `create_latency_plot` isn't used anymore.

Reply:
The latency plot is not used at the moment, but the function remains in case it is needed in the future.